src/Microsoft.Health.Dicom.Api/Web/AspNetCoreMultipartReader.cs

```diff
@@ -90,7 +91,15 @@ internal class AspNetCoreMultipartReader : IMultipartReader
         /// <inheritdoc />
         public async Task<MultipartBodyPart> ReadNextBodyPartAsync(CancellationToken cancellationToken)
         {
-            MultipartSection section = await _multipartReader.ReadNextSectionAsync(cancellationToken);
+            MultipartSection section;
+            try
+            {
+                section = await _multipartReader.ReadNextSectionAsync(cancellationToken);
```

Comment:
Can we catch this exception inside ReadNextSectionAsync.

Generally we should only catch specific exception, otherwise we will catch system exceptions like OutOfMememory exceptions and try to continue to work instead of letting the app fail

Reply:
ReadNextSectionAsync is ASP.NET code but we can catch more specific exception here.

Reply:
I updated it to catch the specific InvalidDataException